gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1-all.zip
```

Comment:
What's the reason for updating the version?

Reply:
The reason is preparation of kotlinx.atomicfu for K2. Expected Gradle version to be compatible with K2 is 8.3, so we decided to steadily update Gradle version first on `develop` and then try out the 8.3-nightly version in the `kotlin-community/k2/dev` branch.

Reply:
Could you please mention that in the commit message when merging the PR? Thanks!